tasties_app/tests/conftest.py

```diff
@@ -183,3 +183,41 @@ def recipe():
 def signed_up_credentials():
     user = User.objects.create_user(username=VALID_USER, email=VALID_EMAIL, password=VALID_PASSWORD)
     user.save()
+
+
+@pytest.fixture
+def recipes_filtered_by_salad():
+
+    """
+    This fixture create the data we need for tests
+    and save it in the data base.
+    """
+    user1 = User.objects.create_user(username="john", password="password")
```

Comment:
This choice of username is a bit risky, since this is actually a reasonable username for a human to choose. Actually, there's already a user "John Doe" that gets created by `0004_create_recipes.py`. Users have a uniqueness constraint on their `username` field. Pick a "weirder" username, like "test_user_1" or even a random meaningless string.

Reply:
is nothing to fair username is UQ name  so if there problem will throw to me exception. also "John Doe " not equal  "John"

Reply:
but anyway I changed.